server/README.md

```diff
@@ -98,7 +98,7 @@
 
 * **URL**
 
-  - `/api/details/`
+  - `/api/details`

```

Comment:
you need to add documentation about what parameters are required when hitting this api

Reply:
you also need to explain the Method and what the route does